### PR TITLE
[3.6] bpo-32356: idempotent pause_/resume_reading (GH-4914)

### PR DESCRIPTION
**Pull request by vstinner**
Backport note: don't add new is_reading() method from master to 3.6.

(cherry picked from commit d757aaf9dd767d13205bf9917e520ebf43e7f6e5)

https://bugs.python.org/issue32356

### CODE REVIEW

File: Doc/library/asyncio-protocol.rst [hunk 1]
```diff
@@ -124,11 +124,19 @@ ReadTransport
       the protocol's :meth:`data_received` method until :meth:`resume_reading`
       is called.
 
+      .. versionchanged:: 3.6.5
```
Comment (ned-deily) [inline Doc/library/asyncio-protocol.rst]: 3.6.5 was already released, and you're going to have to hurry to make 3.6.6!

File: Doc/library/asyncio-protocol.rst [hunk 2]
```diff
@@ -124,11 +124,19 @@ ReadTransport
...
+         The method is idempotent, i.e. it can be called when the
+         transport is already paused or closed.
+
    .. method:: resume_reading()
 
       Resume the receiving end.  The protocol's :meth:`data_received` method
       will be called once again if some data is available for reading.
 
+      .. versionchanged:: 3.6.5
```
Comment (ned-deily) [inline Doc/library/asyncio-protocol.rst]: 3.6.5 -> 3.6.?

**Review by ned-deily — CHANGES_REQUESTED** [review]

File: Doc/library/asyncio-protocol.rst (hunk 2 above)
Comment (vstinner) [reply Doc/library/asyncio-protocol.rst]: Oh. Fixed.

**Comment by vstinner** [comment]
I have made the requested changes; please review again.

**Review by ned-deily — CHANGES_REQUESTED** [review]
The version should be 3.6.7, as it missed the cutoff for 3.6.6,

**Comment by vstinner** [comment]
Oh. I missed the notification about 3.6.6 release (cut off). That's fine, it's a minor issue, no need to push it to 3.6.6.

I will merge this PR once 3.6.6 is released.

**Comment by vstinner** [comment]
I have made the requested changes; please review again.

**Review by ned-deily — APPROVED** [review]

**Comment by vstinner** [comment]
> I will merge this PR once 3.6.6 is released.

I read again @ned-deily's email: changes merged into 3.6 will be released as 3.6.7. So it's fine.

**Pull request merged**